packages/editor/src/components/post-panel-row/index.js

```diff
@@ -15,18 +15,10 @@ const PostPanelRow = forwardRef( ( { className, label, children }, ref ) => {
 			className={ classnames( 'editor-post-panel__row', className ) }
 			ref={ ref }
 		>
-			{ label ? (
-				<>
-					<div className="editor-post-panel__row-label">
-						{ label }
-					</div>
-					<div className="editor-post-panel__row-control">
-						{ children }
-					</div>
-				</>
-			) : (
-				children
+			{ label && (
+				<div className="editor-post-panel__row-label">{ label }</div>
 			) }
+			<div className="editor-post-panel__row-control">{ children }</div>
```

Comment:
The change here is a simplification that also makes the "pending" toggle look better spaced in the post editor.